sonar-db/src/main/java/org/sonar/db/property/PropertiesDao.java

```diff
@@ -170,6 +172,10 @@ public PropertyDto selectProjectProperty(DbSession dbSession, long componentId,
     return executeLargeInputs(keys, partitionKeys -> getMapper(session).selectByKeys(partitionKeys, componentId));
   }
 
+  public List<PropertyDto> selectGlobalPropertiesByKeyQuery(DbSession session, String keyQuery) {
+    return getMapper(session).selectGlobalPropertiesByKeyQuery(buildLikeValue(keyQuery, WildcardPosition.BEFORE));
```

Comment:
why only a WildcartPosition.BEFORE? if it's specifically for licences, then maybe you should rename to make it explicit.
my point is as it is I don't see any reason to have a key query with only a wildcard as a prefix